playwright/src/test/java/com/microsoft/playwright/TestPlaywrightCreate.java

```diff
@@ -41,8 +41,7 @@ void shouldSupportEnvSkipBrowserDownload(@TempDir Path browsersDir) throws IOExc
         getBrowserTypeFromEnv(playwright).launch();
         fail("Did not throw");
       } catch (PlaywrightException e) {
-        assertTrue(e.getMessage().contains("executable doesn't exist"), e.getMessage());
-        assertTrue(e.getMessage().contains(browsersDir.toString()), e.getMessage());
+        assertTrue(e.getMessage().contains("Looks like Playwright Test or Playwright was just installed or updated"), e.getMessage());
```

Comment:
lets follow up on that, to make it cross-language compatible. Sound like a release-blocker or bugfix candidate to me.

Reply:
https://github.com/microsoft/playwright/issues/7741